native-sql-engine/core/src/main/scala/com/intel/oap/execution/ColumnarBasicPhysicalOperators.scala

```diff
@@ -99,7 +99,7 @@ case class ColumnarConditionProjectExec(
     }
   }
 
-  def isNullIntolerant(expr: Expression): Boolean = expr match {
+  override def isNullIntolerant(expr: Expression): Boolean = expr match {
```

Comment:
This impl is as same as the impl in it's parent class, PredicateHelper, in spark 3.2. For keeping the code for spark 3.1 workable, we just changed the method name. For spark3.1. isNullIntolerant is not an interface method or abstract method.